rosdep/base.yaml

```diff
@@ -1166,52 +1166,29 @@ gawk:
   openembedded: [gawk@openembedded-core]
   ubuntu: [gawk]
 gazebo:
-  arch: [gazebo]
   debian:
+    bullseye: [gazebo]
     buster: [gazebo11]
-    jessie: [gazebo7]
-    stretch: [gazebo9]
   fedora: [gazebo]
   gentoo: [sci-electronics/gazebo]
   nixos: [gazebo]
   slackware: [gazebo]
   ubuntu:
-    artful: [gazebo9]
     bionic: [gazebo9]
     cosmic: [gazebo9]
     disco: [gazebo9]
     eoan: [gazebo9]
     focal: [gazebo11]
-    precise: [gazebo]
-    quantal: [gazebo]
-    raring: [gazebo]
-    saucy: [gazebo2]
-    trusty: [gazebo2]
-    wily: [gazebo7]
-    xenial: [gazebo7]
-    yakkety: [gazebo7]
-    zesty: [gazebo7]
+    jammy: [gazebo]
```

Comment:
I believe this will be the key on upstream Ubuntu, and we'll keep releasing `gazebo11` onto packages.osrfoundation.org. So if we decide to use more up-to-date packages, we'll need to change this to `gazebo11`

Reply:
The keys here don't cover upstream packages unless they're part of the bootstrap repository.
So until there are `gazebo11` packages importable for Jammy we can't use an invalid definition.
@j-rivero do the `gazebo11` packages from the OSRF repositories use `Conflicts` or `Replaces` for the packages upstream? I'm trying to figure out what the succession strategy is or if we will need to drop gazebo_ros_packages from Rolling when we first jump to Jammy.

Reply:
There are conflicts here for different versions: https://github.com/ignition-release/gazebo11-release/blob/6492b65efa67c862f8f2d57c58ade9056fe27773/ubuntu/debian/control#L68

We may not have handled the unversioned packages yet since that's going to be a new thing in Jammy.

Reply:
Got it. We should probably add that before this PR goes in.

Reply:
We have the problem of the difference with gazebo naming between our repo and Ubuntu, I'm afraid is not too much I can do about it since Debian prefers to use non-versioned and we need versioned to difference between multiple versions supported.

Debian recommends to handle name changes using `Breaks-Replace` than can be added to our packages to override the Ubuntu packages. Other option is to create a fake transitional package named `gazebo` in our builds of gazebo11 that will pull `gazebo11` and all the friends transparently if the version is greater so we can handle using a single key `gazebo` here.

Reply:
>  Other option is to create a fake transitional package named gazebo in our builds of gazebo11 that will pull gazebo11 and all the friends transparently if the version is greater so we can handle using a single key gazebo here.

I like that. It makes it easier for users who don't know what version they should install :+1:

Reply:
> we'll keep releasing gazebo11 onto packages.osrfoundation.org

I misspoke back then. That's not true for Gazebo classic, see https://community.gazebosim.org/t/gazebo-classic-on-jammy/1248

I believe what we'll need to handle is a conflict between upstream `ligignition-math` and the one provided by packages.o.o if we want users to install Gazebo 11 from Ubuntu alongside Fortress from packages.o.o.